src/Paramore.Brighter.Extensions.DependencyInjection/ServiceCollectionMessageMapperRegistry.cs

```diff
@@ -28,27 +28,23 @@ THE SOFTWARE. */
 using System.Collections.Generic;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.DependencyInjection.Extensions;
+using Paramore.Brighter.MessageMappers;
 
 namespace Paramore.Brighter.Extensions.DependencyInjection
 {
     /// <summary>
     /// When parsing for message mappers in assemblies, stores any found message mappers. A later step will add these to the message mapper registry
     /// Not used directly
     /// </summary>
-    public class ServiceCollectionMessageMapperRegistry
+    public class ServiceCollectionMessageMapperRegistry(
+        IServiceCollection serviceCollection,
+        ServiceLifetime lifetime = ServiceLifetime.Singleton)
     {
-        private readonly IServiceCollection _serviceCollection;
-        private readonly ServiceLifetime _lifetime;
-
         public Dictionary<Type, Type> Mappers { get; } = new Dictionary<Type, Type>();
+        public Type DefaultMessageMapper { get; private set; } = typeof(JsonMessageMapper<>);
```

Comment:
Do we need to default the default here, or do we just do that in the extensions DSLs?

Reply:
I think we need something set if we don't want to make this type nullable, I'm open either way